docs/Maintainer-Guidelines.md

```diff
@@ -23,17 +23,17 @@ Homebrew aims to be the missing package manager for macOS (and Linux). Its prima
 
 1. [Ensure you have set your username and email address properly](https://help.github.com/articles/setting-your-email-in-git/)
 2. Sign off cherry-picks if you amended them (use `git -s`)
-3. If the commit fixes a bug, use “Fixes \#104” syntax to close the bug report and link to the commit
+3. If your commit fixes a bug, use [issue linking syntax](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue) (e.g. “Fixes \#104”) to close the bug report and link back to the commit
 
 ### Add comments
 
-It may be enough to refer to an issue ticket, but make sure changes are clear so that if you came to them unaware of the surrounding issues they would make sense to you. Many times on other projects I’ve seen code removed because the new guy didn’t know why it was there. Regressions suck.
+It may be enough to refer to an issue ticket, but make sure changes are clear so that if you came to them unaware of the surrounding issues they would make sense to you. Many times on other projects I’ve seen necessary code removed because the new guy didn’t know why it was there. Regressions suck.
```

Comment:
I know this was already here, but I find the sudden first person sentence here odd.

Reply:
No way (for me) to know for sure — this passage pre-dates the [wiki-to-Git transition](https://github.com/Homebrew/brew/commit/19d12aee45ac8d55ecd58a8c072286c84e72041d).